src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/CCDCaseApi.java

```diff
@@ -254,6 +269,10 @@ private Claim readCase(User user, String caseId) {
         return search(user, searchString, 1, new ArrayList<>(), null, CaseState.OPEN);
     }
 
+    private List<CaseDetails> searchOnlyClosedCases(User user, Map<String, String> searchString) {
```

Comment:
could we have `searchByCaseState(..., CaseState state)` and pass `CaseState` as argument instead of having two different methods?